wodles/aws/aws_s3.py

```diff
@@ -2394,8 +2394,18 @@ def load_information_from_file(self, log_key):
                 "request_creation_time", "action_executed", "redirect_url", "error_reason", "target_port_list",
                 "target_status_code_list", "classification", "classification_reason")
             tsv_file = csv.DictReader(f, fieldnames=fieldnames, delimiter=' ')
+            tsv_file = [dict(x, source='alb') for x in tsv_file]
+
+            for log_entry in tsv_file:
+                try:
+                    log_entry["client_ip"], log_entry["client_port"] = log_entry["client_port"].split(":")
+                    log_entry["target_ip"], log_entry["target_port"] = log_entry["target_port"].split(":")
+                    log_entry["target_ip_list"], log_entry["target_port_list"] = log_entry["target_port_list"].split(":")
+                except ValueError:
+                    print(f"{WARNING}: Unable to process correctly malformed ABL log entry: {log_entry}.")
```

Comment:
There is no need for a WARNING variable here, as indicated before. This will improve once we replaced `print` statements with `logging` in the future.

Reply:
You should also replace this `print` with our `debug` function which is the one we currently use to display warning and debugging messages. The `print` statements in this module are currently used when an error occurs that will stop execution.

I suggest you to show a warning message with debug 1 and the full log_entry as you proposed with debug 2.

Reply:
I think we should mention in the message that the error is related to the `client_port`, `target_port` or `target_port_list` to avoid confusions.